spago.lock

```diff
@@ -203,7 +218,460 @@ workspace:
         - tuples
       test_dependencies: []
   package_set:
-    registry: 20.0.1
+    address:
+      registry: 20.0.1
+    compiler: ">=0.15.7 <0.16.0"
+    content:
```

Comment:
A bit of a drive-by comment, but why does the lockfile need to contain a full copy of the package set?

Reply:
I answered this in [this comment above](https://github.com/purescript/spago/pull/1138#discussion_r1421788407), let's keep the discussion in that thread